### PR TITLE
fix dead link

### PR DESCRIPTION
**Pull request by mat-de-mystify**
used absolute link as wasn't sure of base of this document

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

File: docs/designers-developers/developers/block-api/block-edit-save.md [hunk 1]
```diff
@@ -406,4 +406,4 @@ When a block is detected as invalid, a warning will be logged into your browser'
 
 **I've changed my block's `save` behavior and old content now includes invalid blocks. How can I fix this?**
 
-Refer to the guide on [Deprecated Blocks](/docs/designers-developers/developers/block-api/block-deprecations.md) to learn more about how to accommodate legacy content in intentional markup changes.
+Refer to the guide on [Deprecated Blocks](https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-deprecation/) to learn more about how to accommodate legacy content in intentional markup changes.
```
Comment (Soean) [inline docs/designers-developers/developers/block-api/block-edit-save.md]:
```suggestion
Refer to the guide on [Deprecated Blocks](/docs/designers-developers/developers/block-api/block-deprecation.md) to learn more about how to accommodate legacy content in intentional markup changes.
```

Good catch! We should use the relative link to get this paged also linked on GitHub.

**Review by Soean — CHANGES_REQUESTED** [review]

**Comment by nosolosw** [comment]
hey @mat-de-mystify this needs some updating before is ready. You'd need to update `block-edit-save.md` with the latest changes and then appyl yours. Is that something you can help with?

File: docs/designers-developers/developers/block-api/block-edit-save.md (hunk 1 above)
Comment (mat-de-mystify) [reply docs/designers-developers/developers/block-api/block-edit-save.md]: cheers for sorting that

**Comment by mat-de-mystify** [comment]
@nosolosw i think thats sorted, bit new to this

**Comment by nosolosw** [comment]
> i think thats sorted, bit new to this

That's fine, we're here to help

**Comment by nosolosw** [comment]
mmm, after the update, I see no changes (only adding a linebreak). I think the fix was already merged in by a different Pull Request that landed before this one. It's a bummer we couldn't get to review this sooner and see your first PR merged. Please, ping me in any future PRs you make and I'll jump to help in a timely manner.

**Pull request closed without merging**